exporter/clickhouseexporter/README.md

```diff
@@ -261,20 +261,29 @@ around 40k/s logs entry per CPU cores, add more collector node can increase line
 
 The following settings are required:
 
-- `endpoint` (no default): The ClickHouse server endpoint, support multi host, for example:
-  tcp protocol `tcp://ip1:port,ip2:port`
-  http protocol `http://ip:port,ip2:port`
+- `endpoint` (no default): The ClickHouse server address, support multi host with port, for example:
+  - tcp protocol `tcp://addr1:port,tcp://addr2:port` or TLS `tcp://addr1:port,addr2:port?secure=true`
+  - http protocol `http://addr1:port,http://addr2:port` or https `https://addr1:port,https://addr2:port`
+  - clickhouse protocol `clickhouse://addr1:port,clickhouse://addr2:port` or TLS `clickhouse://addr1:port,clickhouse://addr2:port?secure=true`
```

Comment:
Is the protocol part supposed to be repeated or typo?
```suggestion
  - tcp protocol `tcp://addr1:port,tcp://addr2:port` or TLS `tcp://addr1:port,addr2:port?secure=true`
  - http protocol `http://addr1:port,addr2:port` or https `https://addr1:port,addr2:port`
  - clickhouse protocol `clickhouse://addr1:port,addr2:port` or TLS `clickhouse://addr1:port,addr2:port?secure=true`
```

Reply:
@StarpTech there are no clickhouse protocol, just http/https or native tcp, `clickhouse://` is same just same to `tcp:// `.

https://github.com/ClickHouse/clickhouse-go/blob/08b27884b899f587eb5c509769cd2bdf74a9e2a1/clickhouse_options.go#L303-L315

Reply:
But they sell it and it works https://github.com/ClickHouse/clickhouse-go#dsn What's your proposal?